File: stringdemo.java
class stringdemo
{
    public static void main(String Args[])
    {
        String s1 = "Hello";
        
        String s2 = new String("Hello");

        char Arr[] = {'H','e','l','l','o'};
        String s3 = new String(Arr);

        System.out.println(s1);
        System.out.println(s2);
        System.out.println(s3);

        System.out.println(s1.length());
        System.out.println(s2.length());
        System.out.println(s3.length());

    }
}

File: stringdemo.c
#include<stdio.h>
#include<string.h>

int main ()
{
    char Arr[] = "hello";
    char Brr[] = {'h','e','l','l','o','\n'};

    printf("%s\n",Arr);
    printf("%s\n",Brr);

    printf("%d\n",strlen(Arr));
    printf("%d\n",strlen(Brr));
    
    return 0;
}